gh2/csv.py

```diff
@@ -245,14 +251,39 @@ def main():
         'Milestone'
     ]
 
-    write_rows(
-        filename=args.output_file,
-        headers=headers,
-        fields=fields,
-        issues=issues_for(repo, state=args.issue_state),
-        date_format=args.date_format,
-        include_prs=args.include_pull_requests,
-        skip_normalization=args.skip_date_normalization,
-        additional_labels=additional_labels,
-        filter_labels=args.filter_labels
+    parser = make_parser()
+    token = os.environ.get('GITHUB_TOKEN')
+    if token is None:
+        parser.exit(status=1,
+                    message='No GITHUB_TOKEN specified by the user\n')
+    args = parser.parse_args()
+
+    filename = args.output_file
+    repositories = args.repositories
```

Comment:
Instead of passing round the repositories, why not make a list of the repo objects here. That save having to pass the token into get_all_labels and saves having to run get_repo in two places.